Massive update

Mer1ck opened the pull request:
Got everything running on 0.G experimental (as of April 11th)
Extended kitsune mutations out to 9 tails, added sprites for the UndeadPeople tileset and WIP support for Ultica.
Mutations now use both mutagen vitamins, and an EOC based system depending on how the player chooses to handle their resources.
Books and items are now properly distributed in item groups, with the fox book occasionally popping up on spell book lists and rarely in mundane bookshelves. Pearls can also be found in wizard's towers and sold at the Forge of Wonders.

I'll continue to be playing with and updating this under my own repo, "FoxMod Revitalized," but I figured I'd dump most of the work here in case you get interest in the mod again.

Mer1ck committed:
Delete dreams.json

Mer1ck committed:
Delete fox magic.json

Mer1ck committed:
Delete kitsune mutations.json

Mer1ck committed:
Delete kitsune spells.json

Mer1ck committed:
Delete magic foxes.json

Mer1ck committed:
Delete modinfo.json

Mer1ck committed:
Delete mutation extensions.json

Mer1ck committed:
Create placeholder.txt

Mer1ck committed:
Add files via upload

Mer1ck committed:
Update README.md

BismuthBorealis commented:
Sounds cool! While I

BismuthBorealis commented:
(Whoops, ignore that misclicked comment post)
Sounds cool! While I probably won't do anything with cdda for a while, if I do it seems likely that with all the changes to the base game since I last worked on this that I'd probably end up going a different direction than you anyway, so I fully support your efforts~
I might end up editing the readme to add a link to your repo/ver if I remember later, too.

Mer1ck commented:
https://github.com/Mer1ck/FoxMod-Revitalized
Here's the link, and thanks for the kind words and support!

Mer1ck committed:
Fixed bugs

Fixed syntax errors in fire/heat armor in the monster entries
Removed "patchfur" and "rough skin" category extensions, and added in the hirsute mutation extension.
